src/OpenSEE/Scripts/TSX/store/dataSlice.tsx

```diff
@@ -547,7 +550,7 @@ export const DataReducer = createSlice({
 });
 
 
-export const { SetHover, SetMouseMode, SelectPoint, SetZoomMode, RemoveSelectPoints, ClearSelectPoints, RemovePlot, UpdateActiveUnits } = DataReducer.actions;
+export const { SetHover, SetMouseMode, SelectPoint, SetZoomMode, RemoveSelectPoints, RemoveLastPoint, ClearSelectPoints, RemovePlot, UpdateActiveUnits } = DataReducer.actions;
```

Comment:
I don't think we need `RemoveLastPoint` here

Reply:
That was a function I had to add specifically for the 'Pop' button. 'RemoveSelectPoints' wasn't working with 'Pop'.

Reply:
Any idea why? That concerns me. maybe we can use RemoveSelectPoints and deal with the difference in the Reducer?

Reply:
Yessir I can go ahead and give it a shot. It was giving me index out of bounds errors when I tried originally if I remember correctly, yet it would work perfectly with the 'Remove" button with no issues. I'll go ahead and see if I can fix that.